Cargo.lock

```diff
@@ -8425,9 +8425,9 @@ dependencies = [
 
 [[package]]
 name = "pallet-xcm"
-version = "8.0.4"
+version = "8.0.3"
```

Comment:
why downgrade `pallet-xcm`? 8.0.4 was a needed patch fix

Reply:
Ah, thanks!  
Looks like i ran the old command, one crate is on that version to i got mixed up.